package.json

```diff
@@ -6,8 +6,8 @@
   "scripts": {
     "lint": "eslint src tests",
     "test": "mocha --compilers js:babel/register tests",
-    "build": "babel --loose all src/translation-assistant.js -o lib/translation-assistant.js",
     "build:add": "git add lib/",
+    "build": "babel --loose all src/translation-assistant.js -o lib/translation-assistant.js && babel --loose all src/math-translator.js -o lib/math-translator.js",
     "watch_test": "mocha --watch --compilers js:babel/register tests",
     "watch_build": "babel --loose all --watch src/translation-assistant.js -o lib/translation-assistant.js"
```

Comment:
Need to figure out how to watch multiple files. Probably easy, I just need to spent some quality time with babel docs.